react/src/components/LegacyFolderExplorer.tsx

```diff
@@ -84,6 +83,9 @@ const LegacyFolderExplorer: React.FC<LegacyFolderExplorerProps> = ({
           ...FolderExplorerHeaderFragment
           ...VFolderNodeDescriptionFragment
           ...VFolderNameTitleNodeFragment
+          # ControlItemsFragment is used in FileExplorer
+          ...BAIFileExplorerFragment
+          ...ControlItemsFragment
```

Comment:
You can remove the `ControlItemsFragment` from this component if it is not directly used here, after PR #4002(merged).